py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -162,6 +159,22 @@ def __setCreds(conf, auth):
         someCreds = JavaWrapper._new_java_obj("scala.Some", creds)
         field.set(jconf, someCreds)
 
+    def __isStopped(self):
+        hc = self._jhc.h2oContext()
+        field = self.__getStoppedField()
+        return field.get(hc)
+
+    def __setStopped(self):
+        hc = self._jhc.h2oContext()
+        field = self.__getStoppedField()
+        field.set(hc, True)
+
+    def __getStoppedField(self):
+        hc = self._jhc.h2oContext()
+        field = hc.getClass().getSuperclass().getDeclaredField("stopped")
+        field.setAccessible(True)
```

Comment:
Just a comment I don't think it's crucial: Shouldn't `__isStopped` and `__setStopped` call `field.setAccessible(False)` when a given operation is finished?

Reply:
I don't think it's necessary as we don't store the reference to the field, but good catch!